modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -26,6 +26,16 @@ def checkoutFromGitHubWithSSH(String repository, String org = 'alphagov', String
 }
 
 /**
+  * Check if the git HEAD is ahead of master.
+  * This will be false for development branches and true for release branches,
+  * and master itself.
+  */
+def isCurrentCommitOnMaster = sh(
```

Comment:
It's most likely me not understanding as I haven't tested this. Will check it to tomorrow.